src/creatures/creature.hpp

```diff
@@ -695,7 +695,7 @@ class Creature : virtual public Thing, public SharedObject {
 	std::string getShader() const {
 		return shader;
 	}
-	void setShader(const std::string &shaderName) {
+	void setShader(const std::string_view &shaderName) {
```

Comment:
You can remove the & as well; passing a std::string_view by reference actually increases overhead. std::string_view is intended to be passed by value since it is simply a lightweight view of the string.